code/_helpers/global_lists.dm

```diff
@@ -127,6 +128,12 @@ var/global/list/unworn_slots = list(slot_l_hand,slot_r_hand, slot_l_store, slot_
 //////////////////////////
 /////Initial Building/////
 //////////////////////////
+/proc/populate_misc_lists()
+	var/list/paths
+	paths = typesof(/obj/item/weapon/robot_module) - /obj/item/weapon/robot_module
+	for(var/path in paths)
+		var/obj/item/weapon/robot_module/M = path
+		GLOB.robot_module_types += initial(M.name)
```

Comment:
That's not conflict with `robot_modules` variable?

Reply:
Having two same meaning variables is confusing.

Reply:
redid it so it uses robot_modules, i will revisit this someday later